src/utils/collections/getItemByOffset.test.ts

```diff
@@ -0,0 +1,40 @@
+import { getItemByOffset } from '@utils/collections/getItemByOffset';
+
+const collection = Array.from({ length: 10 }, (_, i) => `item${i}`);
+
+test('Returns null if the collection has fewer than 1 items', () => {
+	expect(getItemByOffset([], 0, 1)).toBe(null);
+	expect(getItemByOffset([1], 0, 1)).toBe(1);
+	expect(getItemByOffset([1, 2], 0, 1)).toBe(2);
```

Comment:
This assertions looks irrelevant to a test name.
It's better to test different offset values with empty array. Or change test description